public/sass/base/themes/exercism.scss

```diff
@@ -442,6 +441,8 @@ $nav-tabs-justified-link-border-color:            #ddd !default;
 $nav-tabs-justified-active-link-border-color:     $body-bg !default;
 
 //== Pills
+/*pineapple test*/
```

Comment:
Well spotted - it was just a marker that needs to be removed, which I'll do right now!